kafka/src/main/java/com/bazaarvoice/emodb/kafka/JsonPOJOSerde.java

```diff
@@ -19,12 +20,14 @@ public JsonPOJOSerde(Class<T> cls) {
         this._cls = cls;
         _typeReference = null;
         _mapper = new ObjectMapper();
+        _mapper.registerModule(new JavaTimeModule());
```

Comment:
FWIW, I think  this is a pretty awful way to do this. I think we can at minimum make this static in the class instead of initializing a new object mapper in the constructor.